<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bulk Renamer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="5">
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Renamed Files</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QLineEdit" name="prefixEdit">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Enter filename prefix here</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLineEdit" name="dirEdit">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Enter directory...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Files to be renamed</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QListWidget" name="renamedFiles"/>
   </item>
   <item row="6" column="0">
    <widget class="QListWidget" name="srcFiles"/>
   </item>
   <item row="6" column="1">
    <widget class="QProgressBar" name="progressBar">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="renameFilesBtn">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Rename</string>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QPushButton" name="loadFilesBtn">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Load Files</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+L</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
